config.toml

```diff
@@ -19,8 +19,8 @@ home = [ "HTML", "RSS", "JSON"]
   themeVariant = "blue"
 
 [[menu.shortcuts]]
-name = "<i class='fab fa-fw fa-slack-hash'></i> Add to Slack"
-url = "https://slack.com/oauth/authorize?client_id=12637824912.569770799590&scope=commands,bot"
+name = "<i class='fab fa-fw fa-slack'></i> Community Slack"
+url = "https://botkube-slack.herokuapp.com"
```

Comment:
Change this to join.botkube.io
Wait for @vishal-biyani to change DNS mapping.

Reply:
sure.

Reply:
Currently redirection is working. Lets go ahead with redirection for now

Reply:
Made all requested changes.